package/kernel/mac80211/Makefile

```diff
@@ -419,9 +419,15 @@ config-$(call config_package,rsi91x-sdio) += RSI_SDIO
 
 config-$(CONFIG_LEDS_TRIGGERS) += MAC80211_LEDS
 
+C_DEFINES=
+
+ifeq ($(BUILD_VARIANT),smallbuffers)
+	C_DEFINES+= -DCONFIG_ATH10K_SMALLBUFFERS
+endif
+
 MAKE_OPTS:= -C "$(PKG_BUILD_DIR)" \
 	$(KERNEL_MAKE_FLAGS) \
-	EXTRA_CFLAGS="-I$(PKG_BUILD_DIR)/include $(IREMAP_CFLAGS)" \
+	EXTRA_CFLAGS="-I$(PKG_BUILD_DIR)/include $(IREMAP_CFLAGS) $(C_DEFINES)" \
```

Comment:
Do we need the extra variable? Why not simply doing…

```
ifeq ($(BUILD_VARIANT),smallbuffers)
	EXTRA_CFLAGS+= -DCONFIG_ATH10K_SMALLBUFFERS
endif
```

… after the `MAKE_OPTS` definition?

Reply:
If that works, I am happy to do it! :)

Reply:
@rsalvaterra Did I miss something? For me it seems that the config symbol is not defined then I build mac80211?

Reply:
Oh, silly me! `MAKE_OPTS` is being defined with the complete string. The previous version is fine, sorry about the noise. :/

Reply:
Reverted back to previous version. No problem! :)